README.rst

```diff
@@ -31,8 +31,10 @@ contents of the download.
 Requirements
 ------------
 
+Python header files should be installed.
+
 This version of PyStemmer has been tested using Python series 2.6, 2.7, 3.3,
-3.4, 3.5, 3.6 and pypy.  Builds are checked with `travis`_:
+3.4, 3.5, 3.6 3.7 and pypy. Builds are checked with `travis`_:
```

Comment:
a comma is missing here.

Reply:
I've added a commit to address this since the submitter seems unresponsive.